Mlem/App/Views/Pages/ExternalApiInfoView.swift

```diff
@@ -0,0 +1,170 @@
+    /// The local ``ApiClient`` of the model, created using `entityHost`.
+    let entityLocalApi: ApiClient
+    
+    init(api: ApiClient, actorId: URL) {
+        self.fallbackApi = api
+        self.entityActorId = actorId
+        self.entityLocalApi = .getApiClient(for: actorId.removingPathComponents(), with: nil)
+    }
+    
+    var body: some View {
+        VStack {
+            if isLoading {
+                Text("Diagnosing...")
+                    .foregroundStyle(.secondary)
+            } else {
+                ScrollView {
+                    content
+                }
+            }
+        }
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .animation(.easeOut(duration: 0.2), value: isLoading)
+        .background(palette.groupedBackground)
+        .task(loadData)
+        .presentationDetents([.medium])
+        .presentationBackgroundInteraction(.enabled)
+    }
+    
+    @ViewBuilder
+    var content: some View {
+        VStack(spacing: 16) {
+            box(spacing: 16) {
+                if internalFederationStatus?.isAllowed ?? false, externalFederationStatus?.isAllowed ?? false {
+                    Text(
+                        // swiftlint:disable:next line_length
+                        "Your instance and **\(entityLocalApi.host ?? "")** appear to be federating with one another, but we weren't able to access this content from your instance. This could be because the content is new, and hasn't federated to your instance yet. Or, it could be because your instance has purged this content."
+                    )
+                    .padding(.horizontal, AppConstants.standardSpacing)
+                } else {
+                    avatars
+                        .padding(.horizontal, 16)
+                    Text(text)
+                        .multilineTextAlignment(.center)
+                        .padding(.horizontal, 16)
+                }
+            }
+            box(alignment: .leading) {
+                Text("To work around this, we're accessing this content from **\(fallbackApi.host ?? "")** instead of your instance.")
+                    .padding(.horizontal, AppConstants.standardSpacing)
+            }
+            box(alignment: .leading, spacing: 6) {
+                Text("What is Federation?")
+                    .font(.title2)
+                    .fontWeight(.bold)
+                    .padding(.horizontal, AppConstants.standardSpacing)
+                
+                Text(
+                    // swiftlint:disable:next line_length
+                    "Lemmy instances talk to each other so that content can be shared across sites. This is called \"federation\". Instance administrators can choose which other instances they would like their instance to federate with. Some instances will federate with all other instances, except those on a \"block-list\" curated by the administrators. Other instances might only federate to instances on an \"allow-list\"."
```

Comment:
```suggestion
                    "Lemmy instances talk to each other so that content can be shared across sites. This is called \"federation\". Instance administrators can choose which other instances they would like their instance to federate with. Some instances federate with all but a curated \"block-list\" of other instances; other instances might only federate with instances on an \"allow-list\"."
```